### DISCUSSION TITLE
Copilot issues on VSCode

### DISCUSSION THREAD

**Discussion by amazingfly** [community/community#56717] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I just signed up for github copilot.  I installed the extension from vscode.  It never asked me to sign in to github or anything as far as I can tell.  Github Copilot seems to be installed and enabled as far as I can tell in the settings and extensions.  I even see the icon on the bottom right.  But I get no code suggestions as far as I can see.

I hav un installed and re installed VSCode and the extensions but I still get nothing.

Any help would be great :)

Thanks in advance

**Comment by davecheney** [comment]
Hello, I'm sorry you are having difficulties. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

Can you also please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Comment by amazingfly** [comment]
[INFO] [copilotIgnore] [2023-05-30T17:37:05.495Z] inactive
[INFO] [copilotIgnore] [2023-05-30T17:39:24.629Z] inactive

**Comment by davecheney** [comment, marked as answer]
Looking at telemetry I see no request from your client. Can you please try the following;

1. Double check you have vscode, not oss code, or codium, etc, installed.
2. <img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
3. Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

If none of those work, please contact support https://support.github.com/contact

**Comment by amazingfly** [comment]
Well I would bet signing out and back in did the trick, but I am honestly not sure.  I did a whole bunch of stuff trying to get it to work.  But it does work now :P  

Thank you for the help!
